block.go

```diff
@@ -1155,21 +1155,14 @@ func (p *parser) fencedCode(out *bytes.Buffer, data []byte, doRender bool) int {
 		beg = end
 	}
 	var caption bytes.Buffer
-	line := beg
 	j := beg
-	if bytes.HasPrefix(bytes.TrimSpace(data[j:]), []byte("Figure: ")) {
```

Comment:
Initially I thought it was, because it caused crashes that didn't happen before. Afterwards realized that it wasn't directly my changes that caused it. I can remove this if needed.